helper/update_obs_services.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+api_target="https://api.opensuse.org/trigger/runservice"
+api_token=$1
+
+while read -r build_test; do
+    build_test_path=$(dirname "${build_test}")
+    dist=$(echo "${build_test_path}" | cut -f 3 -d/)
+    arch=$(echo "${build_test_path}" | cut -f 2 -d/)
+    project="Virtualization:Appliances:Images:Testing_${arch}:${dist}"
+    package=$(basename "${build_test_path}")
+    curl_target="${api_target}?project=${project}&package=${package}"
+
+    echo "Updating Test: ${package} for: ${project}"
+    curl -H "Authorization: Token ${api_token}" -X POST "${curl_target}"
```

Comment:
This will make sure that failures are not silently lost (https://superuser.com/questions/590099/can-i-make-curl-fail-with-an-exitcode-different-than-0-if-the-http-status-code-i/1626376#1626376)
```suggestion
    curl --fail-with-body -H "Authorization: Token ${api_token}" -X POST "${curl_target}"
```